MekHQ/src/mekhq/campaign/unit/Unit.java

```diff
@@ -1152,6 +1377,16 @@ public double getLightVehicleCapacity() {
         }
         return bays;
     }
+    
+    // Get only bays to which a light tank has been assigned
+    public double getCurrentLightVehicleCapacity() {
+        return lVeeCapacity;
+    }
+    
+    // Used to assign a tank to a bay on a specific transport ship in the TO&E
+    public void setLightVehicleCapacity(double bays) {
```

Comment:
I gotta ask, why double?

Reply:
I was trying to preserve the datatype of capacity as defined in Bay, which is a double when used for Infantry and Cargo bays.